networkit/cpp/graph/GraphTools.cpp

```diff
@@ -74,6 +74,45 @@ node randomNode(const Graph &G) {
     return v;
 }
 
+std::vector<node> randomNodes(const Graph &G, count n) {
+    std::vector<node> selectedNodes;
+    std::vector<bool> alreadySelected(G.numberOfNodes(), false);
+
+    if (n >= G.numberOfNodes()) {
+        for (const auto sample : G.nodeRange()) {
+            selectedNodes.push_back(sample);
+        }
```

Comment:
This can be simplified with `std::insert(selectedNodes.begin(), G.nodeRange().begin(), G.nodeRange().end())`.